### PR TITLE
Adding tools for the test framework

### PR DESCRIPTION
**Pull request by OlehBabenkoo**

### CODE REVIEW

File: e2e/spec.cy.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe('empty spec', () => {
+  it('passes', () => {
+    cy.visit('https://example.cypress.io')
+  })
+})
```
Comment (VadimNastoyashchy) [inline e2e/spec.cy.ts]: duplicate file & folder

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "demoblaze.com",
+  "version": "1.0.0",
+  "description": "Cypress automation project for demoblaze.com",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint . --ext .ts"
```
Comment (VadimNastoyashchy) [inline package.json]: Please add run and open npm script to run tests. 
Also, each npm script must include ts compile checking and lint checking, then run the test.

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+      "target": "es5",
+      "lib": ["es5", "dom"],
```
Comment (VadimNastoyashchy) [inline tsconfig.json]: Should we use "es5" old version?

File: e2e/spec.cy.ts (hunk 1 above)
Comment (OlehBabenkoo) [reply e2e/spec.cy.ts]: Sorry, I didn't understand. Where to duplicate?

Comment (VadimNastoyashchy) [reply e2e/spec.cy.ts]: 2022-10-13_13h26_13

File: tsconfig.json (hunk 3 above)
Comment (VadimNastoyashchy) [reply tsconfig.json]: Also, don't forget to use strict: true

File: package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "demoblaze.com",
+  "version": "1.0.0",
+  "description": "Cypress automation project for demoblaze.com",
+  "main": "index.js",
+  "scripts": {
+    "open": "npx cypress open",
+    "tsc": "npx tsc --project tsconfig.json",
+    "lint": "eslint . --ext .ts",
+    "check":"npm run tsc && npm run lint && npm run open "
```
Comment (VadimNastoyashchy) [inline package.json]: I mean two npm commands to run tests. For example:
"open": "npm run tsc && npm run lint && npx cypress open "
"run": "npm run tsc && npm run lint && npx cypress run "

File: .github/workflows/github-actions-manually.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install node
+        uses: actions/setup-node@v2
+        with:
+           node-version: '14'
+      - name: Install dependencies
+        run: npm install
+      - name: Cypress run
+        run: npx cypress run
```
Comment (VadimNastoyashchy) [inline .github/workflows/github-actions-manually.yml]: Please delete this file.

File: .github/workflows/github-actions-after-push.yml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+        with:
+          ref: develop
+      - name: Install node
+        uses: actions/setup-node@v2
+        with:
+           node-version: '14'
+      - name: Install dependencies
+        run: npm install
+      - name: Cypress run
+        run: npx cypress run
```
Comment (VadimNastoyashchy) [inline .github/workflows/github-actions-after-push.yml]: Change this run: script from "npx cypress run" into npm "run" from package.json file (after you'll add it in an appropriate file).

**Pull request merged**